### PR TITLE
Adds Placeholder Support in Channel and Fixed Custom Colours in Channels

### LINKED ISSUE #20: Placeholders don't work in channels
`factions:
  prefix: '&a[Factions]&r'
  permission: none
  always_appear: false
  format: '%factionsuuid_faction_name% %player% %suffix%&7: '`
  
  The placeholder is downloaded and reloaded, it works with other plugins, and even works in the config for global chat, just not channels

### PR DESCRIPTION
**Pull request by SoRadGaming**
Fixes #20

### CODE REVIEW

**Comment by SoRadGaming** [comment]
I have compiled and tested it, fixed the issue

**Comment by SoRadGaming** [comment]
Now Fixes Issue #22 Compiled and tested, using this build on my server rn

File: src/me/ryandw11/ultrachat/UltraChat.java [hunk 1]
```diff
@@ -81,11 +82,13 @@ public void onEnable() {
...
         }
 
-        if (getServer().getPluginManager().getPlugin("PlaceholderAPI") != null) {
+        if (Bukkit.getPluginManager().isPluginEnabled("PlaceholderAPI")) {
+            Bukkit.getPluginManager().registerEvents(this, this);
             getLogger().info("Hooked into PlaceholderAPI! You can use the place holders!");
             papi = new PAPIEnabled();
         } else {
             papi = new PAPIDisabled();
+            throw new RuntimeException("Could not find Placeholder API! Plugin can not work without it!");
```
Comment (SoRadGaming) [inline src/me/ryandw11/ultrachat/UltraChat.java:91]: Is Placeholder API meant to be a dependency?

**Comment by ryandw11** [comment]
Thank you for spending time on this pull request, but I am taking a different direction with those problems. I am rewriting the plugin to make the code base cleaner and easier to develop for.

**Pull request closed without merging**